src/Aquifer.API/Endpoints/Resources/Content/NeedsTranslation/List/Endpoint.cs

```diff
@@ -0,0 +1,139 @@
+                {ApplySearchQueryFilter(hasSearchQuery, isExactSearch)}
+                {ApplyBookAndChapterFilter(req.BookCode, req.StartChapter, req.EndChapter)}
+                {(getTotalCount ? "" : groupingWithOffset)}
+                """;
+    }
+
+    private static string ApplyLanguageIdFilter(int languageId)
+    {
+        return $"""
+                AND RC.LanguageId = 1 AND RCV.IsPublished = 1 AND NOT EXISTS (
+                    SELECT 1 FROM ResourceContents RCL
+                    WHERE RCL.ResourceId = R.Id AND RCL.LanguageId = {languageId}
+                )
+                """;
+    }
+
+    private static string ApplyParentResourceIdFilter(int? parentResourceId)
+    {
+        return parentResourceId.HasValue ? $"AND R.ParentResourceId = {parentResourceId.Value}" : "";
+    }
+
+    private static string ApplyBookAndChapterFilter(string? bookCode, int? startChapter, int? endChapter)
+    {
+        var verseRange = BibleUtilities.VerseRangeForBookAndChapters(bookCode, startChapter, endChapter);
+        if (verseRange is null)
+        {
+            return "";
+        }
+
+        var startVerseId = verseRange.Value.Item1;
+        var endVerseId = verseRange.Value.Item2;
+
+        return $"""
+                AND (
+                EXISTS (SELECT 1
+                    FROM [VerseResources] AS [v]
+                    WHERE [R].[Id] = [v].[ResourceId] AND [v].[VerseId] >= {startVerseId} AND [v].[VerseId] <= {endVerseId})
+                OR
+                EXISTS (SELECT 1
+                    FROM [PassageResources] AS [p]
+                    INNER JOIN [Passages] AS [p0] ON [p].[PassageId] = [p0].[Id]
+                    WHERE [R].[Id] = [p].[ResourceId] AND [p0].[StartVerseId] >= {startVerseId} AND [p0].[StartVerseId] <= {endVerseId})
+                OR
+                EXISTS (SELECT 1
+                    FROM [PassageResources] AS [p1]
+                    INNER JOIN [Passages] AS [p2] ON [p1].[PassageId] = [p2].[Id]
+                    WHERE [R].[Id] = [p1].[ResourceId] AND [p2].[EndVerseId] >= {startVerseId} AND [p2].[EndVerseId] <= {endVerseId})
+                OR
+                EXISTS (SELECT 1
+                    FROM [PassageResources] AS [p3]
+                    INNER JOIN [Passages] AS [p4] ON [p3].[PassageId] = [p4].[Id]
+                    WHERE [R].[Id] = [p3].[ResourceId] AND [p4].[StartVerseId] <= {startVerseId} AND [p4].[EndVerseId] >= {endVerseId})
+                )
+                """;
+    }
+
+    private static string ApplySearchQueryFilter(bool hasSearchQuery, bool isExactSearch)
+    {
+        if (!hasSearchQuery)
+        {
+            return "";
+        }
+
+        return isExactSearch ? "AND R.EnglishLabel = @SearchQuery" : "AND R.EnglishLabel LIKE @SearchQuery";
+    }
+
+    private static (bool hasSearchQuery, bool isExactQuery) GetSearchParams(string? searchQuery)
+    {
+        var hasSearchQuery = searchQuery is not null;
```

Comment:
The validator is protecting against `searchQuery == ""` but that code is in a different class and could accidentally change without reference to this method.  For defense in depth I suggest also return `hasSearchQuery = false` when `searchQuery` is empty or whitespace.
```suggestion
        var hasSearchQuery = searchQuery is not null && !string.IsNullOrWhiteSpace(searchQuery);
```